### ISSUE TITLE
Applying SSD+inceptionV2 to self provided dataset

### ISSUE THREAD

**Issue by TomPyonsuke** [tensorflow/models#1818]
### Describe the problem
I'm not exactly sure if stackoverflow will be a better place to ask this but I am trying to train SSD+inceptionV2 detector on my own dataset (~200000 clothes images with single bbox) and the loss stopped dropping at around 5. **So i guess my question is, can anybody provide some guidance on what generally needs to be changed to apply this to a random dataset?**  Due to limited computing resources unfortunately I am not able to exhaust every setting... so any suggestions will be greatly appreciated!

Specifically, 
1. Do I need scaling for encoded GT bboxes if I am not using fast-RCNN?
2. Since SSD forward computes classification loss & localization loss in the same network, what's the effect of setting from_detection_checkpoint flag? I would like to unfreeze all parameters and fine tune them all... 

Finally just out of curiosity, why did you choose inceptionV2 as the base network for SSD as opposed to VGG in the original paper? Is it because of a significant performance edge over VGG?

Sorry about making the question so unorganized. Thank you very much!

**Comment by TomPyonsuke** [comment]
I forgot to mention that I am using the sample config file for SSD+inceptionV2. The only thing I have changed is the number of classes.

**Comment by derekjchow** [comment]
1. Scaling should be done the same way between faster-RCNN and SSD models.
2. If you're using our pretrained checkpoints, please leave from_detection_checkpoint to True. This field should be set to False is using a classification checkpoint to bootstrap feature extractor weights. We do not freeze any feature weights when importing from any checkpoint.

The choice for inceptionV2 was somewhat arbitrary. We will release more models with different base networks in the future.

**Comment by humayun** [comment]
Dear TomPyonsuke,

I am trying to retrain the faster_rcnn_resnet101_pets model on pets dataset as explained in this example, but on my local machine instead of google cloud. But I suck with error when running the train.py file. 
Here is my command: 
(tensorflow_1.2)humayun@fkusa8:~/MD_Stuff/tensorflow_1.2/models/models$ python object_detection/train.py --logtostderr --pipeline_config_path=object_detection/samples/configs/faster_rcnn_resnet101_pets.config --train_dir=object_detection/data/faster_rcnn_resnet101_coco_11_06_2017

Can you guide me how are you running your code or retraining the existing model on new dataset. 
Thank you.
-Humayun

**Comment by thess24** [comment]
@TomPyonsuke can you go into a little more detail about what the bounding boxes are bounding exactly and what categories you are trying to predict? Do you have a sample image with bounding boxes for us to look at? The problem may be with using a single bounding box if the images contain other items you are looking to detect.

**Comment by willSapgreen** [comment]
Hello @derekjchow ,

In your comment, 

> This field should be set to False is using a classification checkpoint to bootstrap feature extractor weights.

does classification checkpoint mean [image classification model library](https://github.com/tensorflow/models/blob/master/research/slim/README.md)

Thank you!

**Comment by harshini-gadige** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
